tests/unit/backend/layers/api/test_portal_api.py

```diff
@@ -1185,12 +1071,11 @@ def test__update_collection_same_doi_does_not_update_metadata(self):
 
 class TestCollectionsCurators(NewBaseTest):
 
-    # 🔴 not quite sure why this fails - a non curator should not have access_type of WRITE
```

Comment:
Note that this test (in main) passes with an access type of `WRITE`, hence my comment. I agree that `READ` makes more sense but we shouldn't break the compatibility.

Reply:
Looks like Trent fixed this in main a couple of weeks ago! https://github.com/chanzuckerberg/single-cell-data-portal/pull/3633